### DISCUSSION TITLE
Collector running on Galaxy Watch 4, working, and i can't stop it

### DISCUSSION THREAD

**Discussion by dwallersv** [NightscoutFoundation/xDrip#2233] [Q&A]
My GW4 keeps pairing on it's own with my G6 transmitter, and running  the collector on the watch. This is verified by the fact my watch kept reading and displaying readings, even though my phone wasn't with me most of the day yesterday. 

I shut down the watch and phone this morning, rebooted the phone, and waited until it was collecting readings. Then, booted the watch. It took over readings almost immediately. 

I played around a bunch with the (supposedly non-working) options for collection on the watch, didn't work, so set them back. Ticked these options both from the phone app, then cleared and tried from the "xdrip prefs" app on the watch. Same result.

This was a few days ago. Then I noticed I was getting readings yesterday without my phone. Huh?

Don't know how I did this, and there are other aspects that aren't working – reading history from the watch doesn't sync with the phone when they are connected again, and you have to reboot both to get collection displaying on the phone again (keeps going on the watch). There are some other broken things that are mostly inconveniences, but I'll take it with the problems.

Here's the log from the reboots this morning...

![Screenshot_20220721-101228_xDrip+](https://user-images.githubusercontent.com/729964/180297144-dc9819ca-5e99-48b8-847b-f7084cce1331.jpg)

**Comment by Navid200** [comment]
So, your watch takes over the communication slot that the phone uses and causes the phone to stop working?

How did you enable this feature on the watch?  Can you not disable it the same way you enabled it?

**Comment by dwallersv** [comment]
Don't know what you mean by "slots".

The GW4 is running android (Wear OS), and when you install the xdrip app to get the watchface and connection back to xdrip on your phone, the BG collector service is also there. But it only works on one watch (pebble?). Does not work on the GW4. Or so says anything you can find on line. Didn't work for me either. But I messed around with the settings trying to make it work.

Then I noticed I was getting readings when my phone wasn't with me. Checking logs when I had my phone again, I saw my watch pairing with the dexcom transmitter and taking readings. What's more, the system is stuck in this state, even though the watch collector is turned off in the settings.

**Reply by Navid200** [reply]
A G6 transmitter has 2 communications slots as explained here:  https://navid200.github.io/xDrip/docs/G6_slot.html

If two devices compete on the same slot, only one can connect.  Not much xDrip can do about that.
If your watch is going to use the same slot that your phone wants to use, at least one is going to fail.

**Reply by dwallersv** [reply]
okay.

Doesn't explain why it's working AT ALL????

**Reply by Navid200** [reply]
I personally have no experience with wear.  My interest is to identify xDrip bugs so that they get labeled and tracked.

May be @psonnera can answer your question.
